### PR TITLE
Issue 2572: Janis rendering for OfficialDocumentPage

### PR DESCRIPTION
**Pull request by Niicck**
Ref: cityofaustin/techstack#2572

### CODE REVIEW

File: src/components/PageHeader/_PageHeader.scss [hunk 1]
```diff
@@ -76,3 +76,7 @@
     margin-bottom: $coa-spacing-xlarge;
   }
 }
+
+.coa-PageHeader--official-document {
+
```
Comment (briaguya) [inline src/components/PageHeader/_PageHeader.scss]: does something go here?

File: src/components/Pages/OfficialDocument.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import moment from 'moment-timezone';
+
+const OfficialDocument = ({ document: { id, date, title, authoringOffice, summary, name, link }, bottomBorder, intl }) => {
+  const containerClasses = ["coa-OfficialDocumentPage__container"]
+  if (!!bottomBorder) {
```
Comment (briaguya) [inline src/components/Pages/OfficialDocument.js]: is this something that https://stackoverflow.com/questions/5449872/css-notlast-childafter-selector could work for?

File: src/components/Pages/OfficialDocument.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <span className="coa-OfficialDocumentPage__small-heading">Author:</span> {authoringOffice}
+      </div>
+      <div className="coa-OfficialDocumentPage__small-heading-container">
+        <span className="coa-OfficialDocumentPage__small-heading">Document:</span> <a href={link}>{name}</a>
+      </div>
+    </div>
+  )
+};
+
+OfficialDocument.propTypes = {
```
Comment (briaguya) [inline src/components/Pages/OfficialDocument.js]: yay proptypes! now I'm gonna need to actually use them on my guide page stuff

File: src/components/PageHeader/_PageHeader.scss (hunk 1 above)
Comment (Niicck) [reply src/components/PageHeader/_PageHeader.scss]: nope

File: src/components/Pages/OfficialDocument.js (hunk 2 above)
Comment (Niicck) [reply src/components/Pages/OfficialDocument.js]: yep

**Pull request merged**